Base/Python/tests/test_sitkUtils.py

```diff
@@ -0,0 +1,49 @@
+import slicer
+#import SimpleITK as sitk
+import sitkUtils as su
+
+import unittest
+
+class SitkUtilsTests(unittest.TestCase):
+
+    def runTest(self):
+        self.setUp()
+        self.test_SimpleITK_SlicerPushPull()
+        self.tearDown()
+
+    def setUp(self):
+        """ Download the MRHead node
+        """
+
+        from SampleData import SampleDataLogic
+        SampleDataLogic().downloadMRHead()
+        myNode1 = slicer.util.getNode('MRHead')
+
+        print("Node Name is %s" % myNode1.GetName())
+        self.assertIsNotNone(myNode1.GetName(),"ERROR: Node Name is None !")
+
+        self.myNode1 = myNode1
+
+    def test_SimpleITK_SlicerPushPull(self):
+        """ Try to pull a slicer MRML image, return the SimpleITK,
+        push it to Slicer, and then compare if the image are the same
+        """
+
+        sitkimage = su.PullFromSlicer(self.myNode1.GetName())
```

Comment:
I would add some sanity checks for the SimpleITK image, size, pixel type, direction matrix, and at least a couple pixels.